chains/orchestrator-paras/node/src/cli.rs

```diff
@@ -144,14 +144,6 @@ pub struct RunCmd {
     /// Id of the parachain this collator collates for.
     #[arg(long)]
     pub parachain_id: Option<u32>,
-
-    /// EXPERIMENTAL: This is meant to be used only if collator is overshooting the PoV size, and
-    /// building blocks that do not fit in the max_pov_size. It is a percentage of the max_pov_size
-    /// configuration of the relay-chain.
-    ///
-    /// It will be removed once <https://github.com/paritytech/polkadot-sdk/issues/6020> is fixed.
```

Comment:
We did this in 2503 update:
<img width="1350" height="198" alt="image" src="https://github.com/user-attachments/assets/86b106fa-ab07-4d61-8fa5-744697f9c547" />


Also, this CLI argument can be helpful for [this](https://github.com/paritytech/polkadot-sdk/pull/8040#issuecomment-2754467977).

We might want to add a test for this argument, but not sure if it's a good idea to remove it, this is a fallback argument.

Reply:
If it compiles it means it was not used

Reply:
I added the `id` as otherwise the argument id is same as the variable name and it was conflicting with another argument's id.

Feel free to remove it if not needed.